ocs_ci/ocs/resources/mcg.py

```diff
@@ -773,10 +773,14 @@ def cli_create_bucketclass_over_namespacestores(
             if "ttl" in namespace_policy:
                 cmd += f" --ttl=={namespace_policy['ttl']}"
             self.exec_mcg_cmd(cmd)
+        elif namespace_policy_type == constants.NAMESPACE_POLICY_TYPE_MULTI.lower():
+            cmd += f" --read-resources='{namestores_name_str}'"
+            cmd += f" --write-resource='{namespacestores[0].name}'"
```

Comment:
Using the first NSS as the write resource is fine, but please add a note at the method's docstring to communicate this.

While you're at it, please also document that we're using the default noobaa backingstore as the cache for cache NSS.